specification/dartLangSpec.tex

```diff
@@ -19285,13 +19294,18 @@ \subsubsection{Comments}
 \LMHash{}%
 \IndexCustom{Documentation comments}{documentation comments}
 are comments that begin with the tokens \code{///} or \code{/**}.
-Documentation comments are intended to be processed by a tool that produces human readable documentation.
+Documentation comments are intended to be processed by
+a tool that produces human readable documentation.
 
 \LMHash{}%
-The scope of a documentation comment immediately preceding the declaration of a class $C$ is the instance scope of $C$.
+The scope of a documentation comment immediately preceding
+the declaration of a class $C$ is
+the class scope of $C$.
 
 \LMHash{}%
-The scope of a documentation comment immediately preceding the declaration of a function $f$ is the scope in force at the very beginning of the body of $f$.
+The scope of a documentation comment immediately preceding
+the declaration of a function $f$ is
+the current scope at the very beginning of the body of $f$.
```

Comment:
Actually not true since local variables are in scope in the entire body. It's the scope just *outside* the body. Is that the parameter scope?

Reply:
Argh, this is very old text, present when the spec was added to the SDK repo. ;-)

I guess the intention is that parameters (obviously) must be in scope, but local variables in the function are not. There is no restriction in a DartDoc comment that it can't talk about instance members, so there's no need to worry about the special scope for initializing formals in a constructor, so the formal parameter scope is fine. Using that now!